deploy/scc_endpoint.yaml

```diff
@@ -24,7 +24,7 @@ requiredDropCapabilities:
 runAsUser:
   type: RunAsAny
 seLinuxContext:
-  type: MustRunAs
+  type: RunAsAny
```

Comment:
this should be changed in ocs-operator too

Reply:
Sure, Once it is in.